plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -237,11 +238,12 @@ fu_plugin_thunderbolt_add (FuPlugin *plugin, GUdevDevice *device)
 		fu_device_add_guid (dev, device_id);
 	if (version != NULL)
 		fu_device_set_version (dev, version);
-	fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_UPDATABLE);
 	if (is_host)
 		fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_INTERNAL);
 	if (is_safemode)
 		fu_device_set_metadata_boolean (dev, FU_DEVICE_METADATA_TBT_IS_SAFE_MODE, TRUE);
+	else
+		fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_UPDATABLE);
```

Comment:
Could this go in the `!safemode` block above?

Reply:
`dev `isn't yet defined there, so it seemed to make sense to do all flags in the same place.  I'll move the `dev `definition higher then.